docs/sections/user_guide/cli/tools/config.rst

```diff
@@ -284,229 +278,224 @@ and an additional supplemental YAML file ``values2.yaml`` with the following con
 
   .. code-block:: text
 
-     $ uw config realize --input-file config.yaml --output-format yaml values.yaml --total
-     [2024-04-02T00:53:04]    ERROR Config could not be realized. Try with --values-needed for details.
-     $ echo $?
-     1
-
-* Realization of individual values is all-or-nothing. If a single value contains a mix of renderable and unrenderable variables/expressions, then the entire value remains unrealized. For example, given ``config.yaml`` with contents
-
-  .. code-block:: yaml
-
-     roses: "{{ color1 }} or {{ color2 }}"
-     color1: red
-
-  .. code-block:: text
-
-     $ uw config realize --input-file config.yaml --output-format yaml
-     roses: '{{ color1 }} or {{ color2 }}'
-     color1: red
+     $ uw config realize --input-file config.txt --update-file update.yaml --output-format yaml
+     Cannot deduce format of 'config.txt' from unknown extension 'txt'
 
-* With the ``--dry-run`` flag specified, nothing is written to ``stdout`` (or to a file if ``--output-file`` is specified), but a report of what would have been written is logged to ``stderr``:
+  The format can be explicitly specified  (``config.txt`` is a copy of ``config.yaml``):
 
   .. code-block:: text
 
-     $ uw config realize --input-file config.yaml --output-file realized.yaml --dry-run values1.yaml
-     [2024-01-23T22:31:08]     INFO values:
-     [2024-01-23T22:31:08]     INFO   date: 20240105
-     [2024-01-23T22:31:08]     INFO   empty: null
-     [2024-01-23T22:31:08]     INFO   greeting: Good Night
-     [2024-01-23T22:31:08]     INFO   message: 'Good Night Moon Good Night Moon '
-     [2024-01-23T22:31:08]     INFO   recipient: Moon
-     [2024-01-23T22:31:08]     INFO   repeat: 2
+     $ uw config realize --input-file config.txt --update-file update.yaml --output-format yaml --input-format yaml
+     values:
+       date: 20240105
+       empty: null
+       greeting: Good Night
+       message: Good Night Moon Good Night Moon
+       recipient: Moon
+       repeat: 2
 
-* If an input file is read alone from ``stdin``, ``uw`` will not know how to parse its contents:
+* Similarly, if an input file is read from ``stdin``, ``uw`` will not automatically know how to parse its contents:
 
   .. code-block:: text
 
-     $ cat config.yaml | uw config realize --output-file realized.yaml values1.yaml
+     $ cat config.yaml | uw config realize --update-file update.yaml --output-format yaml
      Specify --input-format when --input-file is not specified
 
-* To read the config from ``stdin`` and realize to ``stdout``:
+  The format can be explicitly specified  (``config.txt`` is a copy of ``config.yaml``):
 
   .. code-block:: text
 
-     $ cat config.yaml | uw config realize --input-format yaml --output-format yaml values1.yaml
+     $ cat config.yaml | uw config realize --update-file update.yaml --output-format yaml --input-format yaml
```

Comment:
```suggestion
     $ cat config.txt | uw config realize --update-file update.yaml --output-format yaml --input-format yaml
```

Same with this one, if that's not what was intended feel free to disregard.

Reply:
I think this one becomes correct after removing the misleading reference to `config.txt` in the preceding commentary, which I'm about to do.

Reply:
Yup definitely agree, I wasn't sure which way you wanted to go on the comments. This works great! Thanks.